contracts/application/management/BaseAppManager.sol

```diff
@@ -52,8 +52,11 @@ contract BaseAppManager is Ownable {
    * @dev Creates a new proxy for the given contract and forwards a function call to it
    * @dev Useful for initializing the proxied contract
    * @param contractName Name of the contract for which a proxy is desired
-   * @param data Data to be sent as msg.data in the low level call. It should include the signature of the function to be called in the implementation together with its parameters, as described in https://solidity.readthedocs.io/en/develop/abi-spec.html#function-selector-and-argument-encoding.
-   @return Address of the new proxy
+   * @param data Data to be sent as msg.data in the low level call. 
+   * It should include the signature of the function to be called in the
+   * implementation together with its parameters, as described in
+   * https://solidity.readthedocs.io/en/develop/abi-spec.html#function-selector-and-argument-encoding.
```

Comment:
I'm sure solc won't parse these lines given they don't have an `@sth` at the beginning. We can use `@dev` here

Reply:
Since these lines are an extension of the `@param` field, which cannot be repeated across lines, we should add a `@dev` saying : "@dev The data argument should include the signature ...".

Reply:
I'm not sure what do you mean here. It seems correct to me to continue the explanation of the @thing on the next line. 
I see they do it plenty on the solidity docs:
https://github.com/ethereum/solidity/blob/106acd9cbb827a39cd9bfd67866dafe0eeeb31de/test/compilationTests/zeppelin/ReentrancyGuard.sol#L18-L22
This is parsed properly by our docs generator:
![screenshot from 2018-05-22 22-52-38](https://user-images.githubusercontent.com/617831/40405453-eb3ff424-5e19-11e8-9bb0-ae26198f5530.png)
And a quick test with `solc Test.sol --devdoc` shows that it parses the multilines correctly:
```
"params" :
      {
        "data" : "Test test"
      }
```